From 42366fe364daee9de6794118e42452c09d873ebf Mon Sep 17 00:00:00 2001
From: chen <1193496683@qq.com>
Date: Tue, 18 Jun 2019 18:55:50 +0800
Subject: [PATCH 2/2] add java

---
 README.md | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/README.md b/README.md
index 7811695..05db819 100644
--- a/README.md
+++ b/README.md
@@ -13,3 +13,7 @@ hello world
 python
 is 
 ok
+
+java
+is
+ok
-- 
2.17.1

